hub.json

```
@@ -27,5 +27,8 @@
         "dbt_netsuite",
         "dbt_mailchimp",
         "dbt_salesforce"
-    ]
+    ],
+    "Montreal-Analytics": [
+        "dbt-snowflake-utils"
+    ],   
```

```suggestion
    ]
```


We should add CI checks that validate the JSON in these files! Just noticed though that there's a stray comma here that we'll need to remove :)